### PR TITLE
[zh-cn] sync concepts/overview/components.md

### PR DESCRIPTION
**Pull request by ty-dc**
[zh-cn] sync concepts/overview/components.md

### CODE REVIEW

File: content/zh-cn/docs/concepts/overview/components.md [hunk 1]
```diff
@@ -236,12 +236,20 @@ saving container logs to a central log store with search/browsing interface.
...
+   * Etcd's official [documentation](https://etcd.io/docs/).
+   * Several [container runtimes](/docs/setup/production-environment/container-runtimes/) in Kubernetes.
+   * Integrating with cloud providers using [cloud-controller-manager](/docs/concepts/architecture/cloud-controller/).
+   * [kubectl](/docs/reference/generated/kubectl/kubectl-commands) commands.
 -->
-* 进一步了解[节点](/zh-cn/docs/concepts/architecture/nodes/)
-* 进一步了解[控制器](/zh-cn/docs/concepts/architecture/controller/)
-* 进一步了解 [kube-scheduler](/zh-cn/docs/concepts/scheduling-eviction/kube-scheduler/)
-* 阅读 etcd 官方[文档](https://etcd.io/docs/)
+了解以下内容：
```
Comment (windsonsea) [inline content/zh-cn/docs/concepts/overview/components.md:248]:
```suggestion
进一步了解以下内容：
```

File: content/zh-cn/docs/concepts/overview/components.md [hunk 2]
```diff
@@ -236,12 +236,20 @@ saving container logs to a central log store with search/browsing interface.
...
+   * [节点](/docs/concepts/architecture/nodes/) 及其与控制平面的通信 [Control Plane](/docs/concepts/architecture/control-plane-node-communication/)。
+   * Kubernetes 中的 [控制器](/docs/concepts/architecture/controller/)。
+   * [kube-scheduler](/docs/concepts/scheduling-eviction/kube-scheduler/)，它是 Kubernetes 的默认调度程序。
+   * Etcd 的官方[文档](https://etcd.io/docs/) 。
+   * Kubernetes 中的几个[容器运行时](/docs/setup/production-environment/container-runtimes/)。
+   * 使用 [cloud-controller-manager](/docs/concepts/architecture/cloud-controller/) 与云提供商进行集成。
+   * [kubectl](/docs/reference/generated/kubectl/kubectl-commands)  命令。
```
Comment (windsonsea) [inline content/zh-cn/docs/concepts/overview/components.md:255]:
```suggestion
   * [节点](/zh-cn/docs/concepts/architecture/nodes/)及其与[控制平面](/zh-cn/docs/concepts/architecture/control-plane-node-communication/)的通信。
   * Kubernetes 中的[控制器](/zh-cn/docs/concepts/architecture/controller/)。
   * Kubernetes 的默认调度程序 [kube-scheduler](/zh-cn/docs/concepts/scheduling-eviction/kube-scheduler/)。
   * etcd 的官方[文档](https://etcd.io/docs/)。
   * Kubernetes 中的几个[容器运行时](/zh-cn/docs/setup/production-environment/container-runtimes/)。
   * 使用 [cloud-controller-manager](/zh-cn/docs/concepts/architecture/cloud-controller/) 与云提供商进行集成。
   * [kubectl](/docs/reference/generated/kubectl/kubectl-commands) 命令。
```

**Review by windsonsea — COMMENTED** [review]
/approve

**Review by my-git9 — APPROVED** [review]
/lgtm

**Pull request merged**